### ISSUE TITLE
Generation of selected graphic symbols

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#264]
```
Unfortunately, most TrueType fonts do not include a range of graphic symbols 
that have traditionally been available in X bitmap fonts (and thus with xterm).
It would be nice to generate some of them - preferably the easy ones...
(like xterm generates VT100 line drawing characters).

(see attached file since pasting them in here doesn't seem to work for most 
either)

Among these, I would appreciate the following most:
- rounded corners (for nice menu borders)
- broken lines (for scrolled menus)
- eighths stair case (for a fine-grained application scrollbar)
- and some nice markers for menus and text:
  check mark, open right arrow, hyphenation point, return symbol
```

Original issue reported on code.google.com by `towom...@googlemail.com` on 17 May 2011 at 8:47

Attachments:
- [missing-symbols.txt](https://storage.googleapis.com/google-code-attachments/mintty/issue-264/comment-0/missing-symbols.txt)

**Comment by kbilleter** [comment]
I've just switched to mintty and missed my xterm font.  A little searching found this page http://stackoverflow.com/questions/3275956/is-there-a-version-of-fixed-6x13-bitmap-font-with-less-whitespace-line-spacing and this font http://triplehelix.org/~mdl/tmp/fixed6x13.zip which look to have what you want.  OTOH, you give up scaling / smoothing which may or may not matter to you.

**Comment by mintty** [comment]
Thanks for the hint, and in case you’re interested in line spacing adjustment, there is the option `RowSpacing`. But no, pixel fonts are not really useful. If at all, I’d rather *draw* a selected set of graphic symbols, like it’s now done for VT100 line drawing graphics (#130).

**Comment by mintty** [comment]
A limited generation solution for the Block Elements (U+2580-U+259F) would be useful as they appear quite broken in many fonts (e.g. U+2583 LOWER THREE EIGHTHS BLOCK higher than U+2584 LOWER HALF BLOCK).

**Comment by mintty** [comment]
Released 2.9.0.
